### DISCUSSION TITLE
Gradients / Ability to add an "offset" to only cover part of the gradient line

### DISCUSSION THREAD

**Discussion by jbtheard** [tailwindlabs/tailwindcss#4906] [Ideas]
<img width="735" alt="Screenshot 2021-07-06 at 14 46 05" src="https://user-images.githubusercontent.com/294065/124610974-e8e1a880-de68-11eb-8bda-2f3018918248.png">

I'd like to reproduce this kind of gradient that do not fully cover the gradient line, in order to better contrast on one end and better fade on the other.

Something in line with
```
.gradient-test {
    background-image: linear-gradient(
      to right bottom, 
      #f18d82 0px, 
      #f18d82 25%, 
      rgba(248, 165, 126, 0.5) 50%,
      rgba(255, 189, 122, 0) 75%, 
      rgba(255, 189, 122, 0) 100%
      )
  }
```

**Comment by michaelschufi** [comment]
Would something like this help?
https://github.com/tailwindlabs/tailwindcss/discussions/3682

nvm, you would need more than just to, from and via.

**Reply by jbtheard** [reply]
that would definitely be of great help indeed!

**Reply by michaelschufi** [reply]
Could you please upvote the linked discussion? I have not yet received any attention of a collaborator 😅
